### PR TITLE
Support scale 

### LINKED ISSUE #718: Scale field in the Inputs tab
- EdgeTX 2.4.0 Flysky Hall gimbals
- Radiomaster TX16S

In the Inputs tab the field "Scale" will only go as high as 100.
Im trying to get it to read from the capacity telemetry RB1C as source with a scale up to 4500mah. This worked just fine in OTX as the first picture below. Second picture is EdgeTX which is stuck at 100.
Converted models from OTX works fine until I change the Scale value ever so slightly, jumps back to 100.

IMG_20210830_154232
IMG_20210912_162849_

### PR DESCRIPTION
**Pull request by eshifri**
Support scale for telemetry sensor. Fix #718

### CODE REVIEW

File: radio/src/gui/colorlcd/model_inputs.cpp [hunk 1]
```diff
@@ -92,6 +92,67 @@ void deleteExpo(uint8_t idx)
...
+      TelemetryItem &telemetryItem = telemetryItems[sensorIndex];
+      if (telemetryItem.isAvailable())  {
+#endif
+        LcdFlags flags = LEFT | COLOR_THEME_PRIMARY1;
+        drawSensorCustomValue(dc, 3, 2, sensorIndex, lastSensorVal, flags);
+      } else {
+        dc->drawText(3, 2, "---", COLOR_THEME_PRIMARY1);
+      }
+    } else {
+      dc->clearClippingRect();
```
Comment (raphaelcoeffic) [inline radio/src/gui/colorlcd/model_inputs.cpp:118]: You should **never** need this! This will effectively impact other things.

Comment (raphaelcoeffic) [inline radio/src/gui/colorlcd/model_inputs.cpp:118]: You should **never** need this! This will effectively impact other things.

**Review by raphaelcoeffic — CHANGES_REQUESTED** [review]

**Review by raphaelcoeffic — APPROVED** [review]
I'm ok with it now. @pfeerick your call!

**Comment by pfeerick** [comment]
Looks like it works perfectly! 😸 Did a test with frsky telemetry and altitude sensor, and was able to set 1.0 scaling so output would move with 1M above and below neutral, and can also set excessively high values, so this *should* fix the issue re: 4500mah calculated sensor scaling...

**Pull request merged**